src/WheelsAndGoods.DataAccess/Repositories/OrdersRepository.cs

```diff
@@ -13,14 +14,43 @@ public OrdersRepository(DatabaseContext databaseContext)
         {
         }
 
-        public async Task<IReadOnlyCollection<Order>> GetOrders()
+        public async Task<IReadOnlyCollection<Order>> GetOrders(FilterOrderModel filterOrderModel)
         {
-            return await Context.Orders
+            var query = Context.Orders
                 .Include(order => order.Customer)
+                .AsQueryable();
+
+            if (filterOrderModel.From != null)
+            {
+                query = query.Where(order => EF.Functions.ILike(
+                    order.From.ToLower(), $"%{filterOrderModel.From.ToLower()}%"));
```

Comment:
ILike is already case insensitive (while Like is case sensitive). The same everywhere with ILike.